src/utils/file_utils.py

```diff
@@ -20,14 +22,25 @@ def read_yaml_file(file_path: str) -> Dict:
     return data
 
 
-def get_files_from_folder_path(path: Path, extension: str = '*') -> Result[List, str]:
+def get_files_from_folder_path(path: Path, extension: str = '*') -> Result[List[str], str]:
     if not path.exists():
         return Err(f"{path} does not exist.")
 
-    if extension[0] != '.':
-        extension = '.' + extension
+    if not extension.startswith('*'):
```

Comment:
Sorry, what do you mean with mixing?

Reply:
I meant it as: An `extension` expects a different kind of format than what a `glob` is.
In any case, do ignore the above comment as it does make sense to me now.

Having said that, I believe the `extension = '*.' + extension` line should be only `*`. E.g.: `extension = f"*{extension}`. Otherwise you'll end up with values like `*..tiff` if somebody does passes as an argument `extension = .tiff`.

Reply:
Yeah, that was the discussion we had in discord about "what we consider an extension" and basically it was without a dot. Hence `'*.' + extension`, as '.tiff' would not be considered a "valid" extension